Source/WebCore/bridge/objc/objc_utility.mm

```diff
@@ -195,8 +195,8 @@ JSValue convertObjcValueToValue(JSGlobalObject* lexicalGlobalObject, void* buffe
     switch (type) {
         case ObjcObjectType: {
             id obj = *(const id*)buffer;
-            if ([obj isKindOfClass:[NSString class]])
-                return convertNSStringToString(lexicalGlobalObject, (NSString *)obj);
+            if (auto *str = dynamic_objc_cast<NSString>(obj))
```

Comment:
shouldn't all of these new `auto *`s be RetainPtr's?

Reply:
Done that.